integration/apps/minife/minife.py

```diff
@@ -56,12 +55,20 @@ def __init__(self, num_nodes):
             raise RuntimeError("No input size defined for minife on {} nodes".format(num_nodes))
         self.app_params = problem_sizes[num_nodes]
 
+        app_cores = apps.get_available_app_cores(mach, pin_config)
+        # TODO: test other thread configs
+        self.cpu_per_rank = 1
+        self.ranks_per_node = app_cores // self.cpu_per_rank
```

Comment:
So before, ranks_per_node was 2.

Now, I think it's going to be a lot more.  Thinking about 44 cores on McFly:
```
(44 - 2) // 1 = 42
```

Do you want 42 ranks per node?

Reply:
I still need to test miniFE with different rank configurations.  I don't know why we thought 2 ranks was the best, but the documentation I could dig up was using 1 rank per core.  Given that this version has OMP threading, I doubt that MPI threads only is the right configuration.